### PR TITLE
Jignparm/csharp test difftypes

### PR DESCRIPTION
**Pull request by jignparm**
Added tests for pretrained models in C# library

### CODE REVIEW

File: tools/ci_build/github/linux/run_build.sh [hunk 1]
```diff
@@ -20,7 +20,7 @@ if [ -z "$AZURE_BLOB_KEY" ]; then
 else
   echo "Downloading test data from azure"
   mkdir -p /home/onnxruntimedev/models/
-  azcopy --recursive --source:https://onnxruntimetestdata.blob.core.windows.net/onnx-model-zoo-20181018 --destination:/home/onnxruntimedev/models/ --source-key:$AZURE_BLOB_KEY
+  azcopy --recursive --source:https://onnxruntimetestdata.blob.core.windows.net/onnx-model-zoo-20181206 --destination:/home/onnxruntimedev/models/ --source-key:$AZURE_BLOB_KEY
```
Comment (shahasad) [inline tools/ci_build/github/linux/run_build.sh]: so the version 20181206 includes the newly created tiny test models?

File: tools/ci_build/github/azure-pipelines/win-gpu-ci-pipeline.yml [hunk 2]
```diff
@@ -19,7 +19,7 @@ jobs:
       displayName: 'Get ONNX testdata'
       inputs:
         filename: azcopy
-        arguments: ' /S /Source:https://onnxruntimetestdata.blob.core.windows.net/onnx-model-zoo-20181018 /Dest:$(Build.SourcesDirectory)\build\Windows\Debug\models /SourceKey:%AZURE_BLOB_KEY%'
+        arguments: ' /S /Source:https://onnxruntimetestdata.blob.core.windows.net/onnx-model-zoo-20181206 /Dest:$(Build.SourcesDirectory)\build\Windows\Debug\models /SourceKey:%AZURE_BLOB_KEY%'
```
Comment (shahasad) [inline tools/ci_build/github/azure-pipelines/win-gpu-ci-pipeline.yml]: Why did we need a new version of the models?

Comment (jignparm) [reply tools/ci_build/github/azure-pipelines/win-gpu-ci-pipeline.yml]: Removed this dependency now. Reading directly from PB files

File: csharp/src/Microsoft.ML.OnnxRuntime/Microsoft.ML.OnnxRuntime.csproj [hunk 3]
```diff
@@ -82,6 +82,7 @@
   </Target>
 
   <ItemGroup>
+    <PackageReference Include="Google.Protobuf" Version="3.6.1" />
```
Comment (shahasad) [inline csharp/src/Microsoft.ML.OnnxRuntime/Microsoft.ML.OnnxRuntime.csproj]: remove this dependency from src, and make it a dependency of the test project

Comment (shahasad) [reply csharp/src/Microsoft.ML.OnnxRuntime/Microsoft.ML.OnnxRuntime.csproj]: Same for the OnnxML.cs

Comment (jignparm) [reply csharp/src/Microsoft.ML.OnnxRuntime/Microsoft.ML.OnnxRuntime.csproj]: done.

**Pull request merged**